build.rs

```diff
@@ -69,8 +70,7 @@ fn build_vendored_sundials() -> (Option<String>, Option<String>, &'static str) {
     (lib_loc, inc_dir, library_type)
 }
 
-fn generate_bindings(inc_dir: &Option<String>)
-                     -> Result<Bindings, BindgenError> {
+fn generate_bindings(inc_dir: &Option<String>) -> Result<Bindings, BindgenError> {
```

Comment:
Could you please refrain from modifying code that you do not change?

Reply:
That change was consistent with rustfmt, but has been reverted